src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -294,11 +300,15 @@ def _spawn(self, distribution, executor=None, *args, **kwargs):
                                            executor=actual_executor,
                                            **kwargs)
 
+  @deprecated(removal_version='1.6.0.dev0',
+              hint_message='Subclasses should calculate the correct java distribution to use and '
```

Comment:
Out of curiosity, why not just kill the method? It's not marked as public. Are there known users?